tiledb/sm/group/group.cc

```diff
@@ -287,18 +277,15 @@ const shared_ptr<GroupDetails> Group::group_details() const {
   return group_details_;
 }
 
-Status Group::get_query_type(QueryType* query_type) const {
+QueryType Group::query_type_checked() const {
```

Comment:
`query_type`. The annotation `checked` doesn't add any information. And, insofar as I can tell, this function does no checking itself (maybe deeper in the call stack). There's no parallel `unchecked` version I can see.

Reply:
It throws if the group is not open. There is an unchecked `query_type()` method in line 500.